integration-templates/anrok/utils.ts

```diff
@@ -0,0 +1,9 @@
+export function errorToObject(err: unknown) {
+    if ('response' in (err as any)) {
+        return (err as any).response.data;
+    }
```

Comment:
would it make sense to use `isAxiosError`? here we assume that if there is `response` field we return `data` but there is no guarantuee

Reply:
we don't allow import axios, maybe that's something we should change or expose in the sdk
(imo we will get rid of axios at some point so I wouldn't bother)

Reply:
let me know if you prefer to have this fixed rn 👍🏻

Reply:
don't spend too much time on that. it is fine for now